### ISSUE TITLE
Email Error that the customer will see.

### ISSUE THREAD

**Issue by maurice6392** [abantecart/abantecart-src#1570]
This Error is what the customer see's after completing and order:
**Unknown error encountered. Please try again!**
But the order goes through and sends the email to the customer... but if they go by the cart screen, they will think their order did not go through and may repeat the order multiple times... which would be a major pain for them and me.!
This is the error from the Error logs:
2023-03-13 21:08:08 - Unknown Error: AbanteCart core v.1.3.3 Email "errors" does not comply with addr-spec of RFC 2822. in <b>/home/lrtr9wz13wxw/public_html/PEGASUSADULTSTORE.COM/vendor/symfony/mime/Address.php</b> on line <b>56</b>

I have tried all email option that it has and this is the only one that will actually send the email out but the customer gets an Error as described above.
This is the email settings page as the cart defaults to:
![image](https://user-images.githubusercontent.com/126823484/224836997-ed8f0d2b-012c-4803-a9e8-4ca9f3c273ac.png)

Thank you in advance!

**Comment by BasaraABC** [comment]
Please remove the "errors," word from the Additional emails field

**Comment by maurice6392** [comment]
I did remove it and tested it 3 additional times, on the customer side the error still shows.!
I removed the word  error,  right after I posted the problem to Github
problem is still there.
   On Tuesday, March 14, 2023 at 02:13:08 AM EDT, BasaraABC ***@***.***> wrote:  
 
 


Please remove the "errors," word from the Additional emails field

**Comment by maurice6392** [comment]
I just tried it again and the error is gone, but it was still showing at 6:00 Pm. EDT.

Why did it take ten hours, it should have been immediate after I reset the server 

I hope I have no more errors.

Thanks

   On Tuesday, March 14, 2023 at 02:13:10 AM EDT, BasaraABC ***@***.***> wrote:  
 
 


Closed #1570 as completed.

**Comment by BasaraABC** [comment]
Hello. You don't need to reset a server when you change the AbanteCart settings.

**Comment by maurice6392** [comment]
Hello,I did have to reset the server with the GoDaddy  installatron Application install, but not everything but some required it to show the changes.

the Abantecart at GoDaddy needs to be updated and fixed as anyone who uses it will have the Authorize.net problem like I did...

Question: is there any way for Abantecart to allow transparent images on like the logo's that I created, it made the background white.
So I had to make my logo to match the template as it does not allow transparent images.


Thank you

 Maurice

   On Tuesday, March 14, 2023 at 04:33:41 AM EDT, BasaraABC ***@***.***> wrote:  
 
 


Hello. You don't need to reset a server when you change the AbanteCart settings.
